recipe/meta.yaml

```diff
@@ -1,59 +1,155 @@
 {% set name = "dbt-snowflake" %}
-{% set version = "1.7.0" %}
+{% set version = "1.8.4" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  - url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/dbt-snowflake-{{ version }}.tar.gz
-    sha256: 959943cdb0bcedc67203f0205a9ae21fba13f4c95ddf7089848a77b8c8a8d5f1
-
-  - url: https://github.com/dbt-labs/dbt-snowflake/archive/refs/tags/v{{ version }}.zip
-    sha256: 01e450dd7b6737a96e40d50bca25aa02ea2f69f6f274b43f7c80ea7cccb737dc
-    folder: dbt-snowflake-git
+  url: https://github.com/dbt-labs/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 80f4f6ab5b8f40a36cbf30b295df4fb68786624db88b875f56d8b8c813664ae1
 
 build:
-  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation --ignore-installed --no-cache-dir -vv
   number: 0
-  # Skip s390x because we don't have agate on s390x.
-  skip: true  # [py<38 or s390x]
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  skip: True  # [py<39 or py>311]
```

Comment:
https://github.com/dbt-labs/dbt-snowflake/blob/v1.8.4/setup.py#L81 suggests `[py<38]`.  This way our recipe reflects what the upstream says (even if we don't build for Python 3.8 ourselves).

I'd be less concerned that the highest classifiers s Programming Language :: Python :: 3.11", I'd not use an upper bound and see if it builds (and tests).